ppr-ui/src/components/registration/RegistrationLengthTrust.vue

```diff
@@ -176,4 +234,55 @@ export default defineComponent({
 .length-trust-label {
   font-size: 0.875rem;
 }
+
+.length-trust-summary-header {
+  display: flex;
+  background-color: $gray3;
+  padding: 1.25rem;
```

Comment:
I don't know what the best practice is, but I generally find it easier/simpler to adjust things on the UI when most of the padding/margins are defined in the html with the `p/m..-..` classes. I see you've defined padding in here and with `pa-2` in the html haha. Can you consolidate them so the padding is defined in one place for this class and any other ones that do this? If there isn't a pa-x value that works then just having the value here is good I think